src/pages/contributing/components.mdx

```diff
@@ -87,3 +88,125 @@ For guidance, see our Documentation requirements section above or reach out to u
 
+## How to write component guidance
+
+Our component documentation typically contains five sections: Default, Usage guidance, Design and functional specifications, Development documentation, and Feedback. Some components are more complex than others but you should cover each topic included in these markdown templates.
+
+#### Template for components with one variant
+Use this template for documenting components with only one variant. 
+
+```markdown
+---
+title: Component name
+description: Explains what the component is in one or two sentences.
+---
+
+import { ComponentStatus } from 'components/ComponentList';
+import ComponentDescription from 'components/ComponentDescription';
+import ComponentFeedback from 'components/ComponentFeedback';
+
+<ComponentDescription name="Component nam" type="ui or ayout" />
+
+<AnchorLinks>
+
+<AnchorLink>Default</AnchorLink>
+<AnchorLink>Usage guidance</AnchorLink>
+<AnchorLink>Design and functional specifications</AnchorLink>
+<AnchorLink>Development documentation</AnchorLink>
+<AnchorLink>Feedback</AnchorLink>
+
+</AnchorLinks>
+
+## Default
+
+<!--
+Include a paragraph or two that describes the primary use cases for the component and details about how it helps the user achieve any tasks.
+
+-->
+
+![A visual of Component variation (URL or path to the image)
+
+## Usage guidance
+
+<!--
+Include a paragraph or two that describes the format and what behaviors the component should be.
+
+-->
+
+## Design and functional specifications
+
+The design specs and functional specs for “Component nam” can be viewed  <a href=“Link “to the BOX folder target="_blank">here</a>.
+
+## Development documentation
+
+<ComponentStatus name="Component nam" type=“ui or layout" />
+
+## Feedback
+
+<ComponentFeedback />
+```
+
+#### Template for components with multiple variants
+Use this template for documenting components that have multiple types or
+variants.
+
+```markdown
+---
+title: Component name
+description: Explains what the component is in one or two sentences.
+---
+
+import { ComponentStatus } from 'components/ComponentList';
+import ComponentDescription from 'components/ComponentDescription';
+import ComponentFeedback from 'components/ComponentFeedback';
+
+<ComponentDescription name="Component nam" type="ui or ayout" />
+
+<AnchorLinks>
+
+<AnchorLink>Default</AnchorLink>
+<AnchorLink>Variations</AnchorLink>
+<AnchorLink>Usage guidance</AnchorLink>
+<AnchorLink>Design and functional specifications</AnchorLink>
+<AnchorLink>Development documentation</AnchorLink>
+<AnchorLink>Feedback</AnchorLink>
```

Comment:
Need a comment here on how each section below gets a link here, and how it has to be the same exact text.

Reply:
Like...

<!--
The links represent the section titles below. Please make sure the name listed is the exact text as the section title below.
-->

Reply:
Sorry, like what?

```markdown
<!--
Include additional <AnchorLinks> if you add additional sections within the page. Text must match header text.
<AnchorLink>Header text you're linking to</AnchorLink>
-->
```

Reply:
Got it..